src/Components/Agenda/AgendaItemMaintenanceContainer/AgendaItemMaintenanceContainer.jsx

```diff
@@ -151,6 +150,14 @@ const AgendaItemMaintenanceContainer = (props) => {
     dispatch(modifyAgenda(maintenanceInfo, legs, personId, efInfo, agendaItemData$));
   };
 
+  const removeAI = () => {
+    const data = {
+      aiseqnum: get(agendaItemData$, 'pmi_seq_num'),
+      aiupdatedate: get(agendaItemData$, 'pmi_update_date'),
```

Comment:
```suggestion
      aiseqnum: agendaItemData$?.pmi_seq_num
      aiupdatedate: agendaItemData$?.pmi_update_date
```

quick, before Moose notices this Lodash get 😝

Reply:
@elizabeth-jimenez haha ah yea he hates it! Changed!